pkg/api/api.go

```diff
@@ -29,9 +29,10 @@ func NewAPI(routes *routes.Routes) *API {
 }
 
 type Config struct {
-	StorageDriver string
-	Version       string
-	UseScopes     bool
+	StorageDriver     string
+	Version           string
+	UseScopes         bool
+	RedisLockStrategy bool
```

Comment:
Why needed here?

Reply:
Oops, I tried different things to pass down the strategy, and forgot to remove this one thanks !